### PR TITLE
Add TBH support

### PR DESCRIPTION
**Pull request by duc996**
This pull request adds support for devices from the tbh project (manufactured by archos).

### CODE REVIEW

**Comment by zuckschwerdt** [comment]
Looks good, thanks! I'll review shortly. Do we have sample files in https://github.com/merbanan/rtl_433_tests ?

**Comment by duc996** [comment]
I have just added a sample file and created the pull request.

**Comment by duc996** [comment]
Sorry for the delay, I have added a sample file just after receiving your message. I have added a comment on github.

----- Original Message -----

File: src/devices/tbh.c [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+  "timestamp",
+  NULL,
+};
+
+r_device tbh = {
+  .name = "TBH weather sensor",
+  .modulation = FSK_PULSE_PCM,
+  .short_width = 212,
+  .long_width = 212,
+  .reset_limit = 217088,
```
Comment (merbanan) [inline src/devices/tbh.c]: Reset limit is high.

File: src/devices/tbh.c [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    }
+
+    uint16_t crc;
+    crc = crc16 (frame, len + 1, 0x8005, 0xffff);
+
+    if (decoder->verbose)
+      fprintf (stderr, "got CRC %04x\n", crc);
+
+    if ((frame[len + 1] << 8 | frame[len + 2]) != crc) {
+      if (decoder->verbose) {
```
Comment (merbanan) [inline src/devices/tbh.c]: MIC maybe should be added to the output.

**Comment by zuckschwerdt** [comment]
And Indent is 2 instead of 4. I'll just merge shortly and the clean it up.

**Comment by zuckschwerdt** [comment]
Clean up in 8ed6eaf. Note that the single sample we have shows a truncated preamble.

**Pull request merged**